install/troubleshooting/rds_pg_stat_statements_shared_preload_libraries.md

```diff
@@ -1,21 +0,0 @@
----
-title: 'Amazon RDS: Resolving the "pg_stat_statements must be loaded via shared_preload_libraries" error'
```

Comment:
As mentioned in Slack, I think we should keep this page for now (and may want to make it more generic in the future, so it doesn't just talk about RDS, but rather in general addresses this setup error)